### ISSUE TITLE
Rocksdb build fails

### ISSUE THREAD

**Issue by kk00ss** [pmwkaa/ioarena#2]
I've managed to build Rocksdb locally in a different directory, but ioarena build script fails with

```
db/compaction_job.o: In function `rocksdb::CompactionJob::Run()':
/usr/include/c++/4.9/thread:136: undefined reference to `pthread_create'
```

Which is strange.

**Comment by pmwkaa** [comment]
Now sure how can i help here.
Try to remove cmake cache files and rebuild again:

``` sh
rm -rf CMakeFiles CMakeCache.txt
cmake -DENABLE_ROCKSDB=ON .
make
```

**Comment by kk00ss** [comment]
Can I just copy binaries into some folder and configure ioarena to use them ?

**Comment by pmwkaa** [comment]
CMake script tries to find first a library installed in system, if it fails then make an attempt to build a shipped one.

https://github.com/pmwkaa/ioarena/blob/master/CMakeLists.txt#L32

**Comment by kk00ss** [comment]
I've executed make install for multiply databases - still build script builds them all. I've tried to change find_package to find_library - same result. But thank you for such a tool. As for LMDB - there is a way to speedup writes by using multiply environments (different B-trees so write lock on one of them doesn't stop all the write operations - but I don't know whether it is possible to create transactions that will use multiply environments) - by sharding operations between them using key-prefix. I'll do separate tests of RocksDB.
